geniza/corpus/tests/test_corpus_models.py

```diff
@@ -208,6 +208,12 @@ def test_str(self):
         doctype = DocumentType(name="Legal")
         assert str(doctype) == doctype.name
 
+    def test_display_label(self):
+        legal = DocumentType(name="Legal", display_label="Legal document")
+        assert legal.display_label == "Legal document"
+        no_label = DocumentType(name="test")
+        assert not no_label.display_label
```

Comment:
What are you testing here?

What you need to test is the new case for `str` when `display_label` is set